### ISSUE TITLE
Need to comment out the cli to use the proxy on windows

### ISSUE THREAD

**Issue by yschimke** [whyoleg/rkncli#2]
I guess failing here from the build file

```
        HostManager.hostIsLinux -> linuxX64("native")
        HostManager.hostIsMac   -> macosX64("native")
        else                    -> error("not supported")
```

Is windows support easy to add?

**Comment by yschimke** [comment]
Trivial fix e.g 

```
        HostManager.hostIsMingw -> mingwX64()
```

Doesn't work because of dependencies

**Comment by whyoleg** [comment]
ktor doesn't support TCP on K/N on windows (not sure why), so rsocket-kotlin also doesn't support it

**Comment by yschimke** [comment]
Can I make the cli conditionally loaded on windows? Or is there a way to target this to build with JDK?

**Comment by whyoleg** [comment]
for now just not include `cli` module in settings.gradle
I will update implementation soon to have also JVM TCP target in `cli` project - it's easy enough

**Comment by yschimke** [comment]
Yep, no rush, I've done that.  Mainly found the proxy really useful for debugging.
